tools/decode_jwt_payload.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python3
```

Comment:
hey @JohanLorenzo, by chance would you have the bandwidth to review this simple python script with the corresponding test for me? Thanks a lot :-)

Reply:
nevermind @johanlorenzo, our nice contributor @brisad offered to help on the review!

Reply:
👍 

I'm sorry, I hadn't had the time to look at it with the current all-hands.